### PR TITLE
 added foxlon11a and foxlon11b.

### PR DESCRIPTION
**Pull request by EtienneTignon**
Sorry for the pull requests on the SonsRecommendation branch. I discussed with Javier this morning and I hope by starting anew I will do things correctly this time. If this is correct, I'll add the other papers I should.

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -9048,6 +9048,24 @@ @article{foxlon03a
...
+}
+
+@article{foxlon11b,
+  title = {{PDDL2.1}: An Extension to {PDDL} for Expressing Temporal Planning Domains},
+  author = {M. Fox and D. Long},
+  journal = {CoRR},
+  url = {http://dblp.uni-trier.de/db/journals/corr/corr1106.html#abs-1106-4561},
+  volume = {abs/1106.4561},
+  year = {2011}
+}
```
Comment (tortinator) [inline krr.bib:9067]: ditto https://www.jair.org/index.php/jair/article/view/10352

**Review by tortinator — CHANGES_REQUESTED** [review]
@EtienneTignon , `arxiv` publications are only used as a transition, the ultimate paper should always be cited *if already available*

File: krr.bib (hunk 1 above)
Comment (EtienneTignon) [reply krr.bib]: It was removed, I found the paper somewhere else on the document.

File: krr.bib [hunk 2]
```diff
@@ -9048,6 +9048,24 @@ @article{foxlon03a
...
 }
 
+@article{foxlon11a,
+  title = {Modelling Mixed Discrete-Continuous Domains for Planning},
+  author = {M. Fox and D. Long},
+  journal = {CoRR},
+  url = {http://arxiv.org/abs/1110.2200},
+  volume = {abs/1110.2200},
+  year = {2011}
+}
```
[the remark this answers is not in the document]
Comment (EtienneTignon) [reply krr.bib]: I corrected the date and linked to the definitive(?) version of the paper.
I think it should be ok now.

File: krr.bib [hunk 3]
```diff
@@ -9048,6 +9048,15 @@ @article{foxlon03a
...
 
+@article{foxlon06a,
+  title = {Modelling Mixed Discrete-Continuous Domains for Planning},
+  author = {M. Fox and D. Long},
+  journal = jair,
+  url = {https://dl.acm.org/doi/10.5555/1622572.1622580},
+  volume = {27},
+  year = {2006}
+}
+
```
Comment (tortinator) [inline krr.bib:9059]: You miss the page numbers and the issue though

**Review by tortinator — CHANGES_REQUESTED** [review]
Why do you think page numbers and issues are irrelevant..?

File: krr.bib (hunk 3 above)
Comment (EtienneTignon) [reply krr.bib:9059]: I didn’t knew if it was relevant. I looked at other entries and followed them.
I will add them.

**Review by tortinator — APPROVED** [review]

**Pull request merged**